src/agent/base_agent.py

```diff
@@ -116,7 +118,8 @@ def __init__(
 
         self.debug = debug
         self.routing_agent = ReActAgent.from_tools(
-            [search_tool, locate_tool, vision_tool], llm=llm, verbose=True
+            # [search_tool, locate_tool, vision_tool], llm=llm, verbose=True
+            [search_tool, locate_tool,], llm=llm, verbose=True
```

Comment:
Maybe we can make these tools with booleans to choose selectively.
Could you please add an argument for tool selection?

Reply:
pr #26